test/event-manager.spec.js

```diff
@@ -120,6 +128,20 @@ describe('EventManager', () => {
       threeClick.calls.reset();
     });
 
+    it('bubbles properly out of shadow dom when not delegated with composed flag', () => {
```

Comment:
tests are good 👍

Reply:
Thanks, I also corrected a potential pitfall. Moved the creation of the elements into the before each, the previous setup would append the same elements over and over again. If the tests would have checked on the amount of delegate calls I would expect it to have failed with the previous setup.

Reply:
Thanks 😎